rspec-tools/tests/validation/test_description_validation.py

```diff
@@ -110,7 +110,7 @@ def test_single_how_to_fix_it_allowed_validation(invalid_rule):
 def test_duplicate_frameworks_in_how_to_fix_it_validation(invalid_rule):
   '''Check that duplicate "How to fix it" subsections for the same framework breaks validation'''
```

Comment:
_minor_ we could also update the test name and description for clarity.
```suggestion
def test_duplicate_h2_sections_validation(invalid_rule):
  '''Check that duplicate H2 sections breaks validation'''
```